Homework1/Sort/Sort/BubbleSort.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace Sort;
+public class BubbleSort
+{
+    public static void BubbleSortArray(int[] arrayOfNumbers)
+    {
+        for (int i = 0; i < arrayOfNumbers.Length - 1; i++)
+        {
+            for (int j = arrayOfNumbers.Length - 1; j > i; j--)
+            {
+                if (arrayOfNumbers[j - 1] > arrayOfNumbers[j])
+                {
+                    (arrayOfNumbers[j - 1], arrayOfNumbers[j]) = (arrayOfNumbers[j], arrayOfNumbers[j - 1]);
+                }
+            }
+        }
+    }
+    public static void Main(string[] args)
```

Comment:
```suggestion
    }
    
    public static void Main(string[] args)
```
Надо пустые строки между методами. Стайлгайд :)